builder/proxmox/step_type_boot_command_test.go

```diff
@@ -52,6 +52,20 @@ func TestTypeBootCommand(t *testing.T) {
 			expectedKeysSent:  "shift-hellospcshift-worldspc2dot0fooshift-1barshift-2baz",
 			expectedAction:    multistep.ActionContinue,
 		},
+		{
+			name:              "holding and releasing keys",
+			builderConfig:     &Config{BootConfig: bootcommand.BootConfig{BootCommand: []string{"<leftShiftOn>hello<rightAltOn>world<leftShiftOff><rightAltOff>"}}},
+			expectCallSendkey: true,
+			expectedKeysSent:  "shift-hshift-eshift-lshift-lshift-oshift-alt_r-wshift-alt_r-oshift-alt_r-rshift-alt_r-lshift-alt_r-d",
+			expectedAction:    multistep.ActionContinue,
+		},
+		{
+			name:              "holding multiple alphabetical keys and shift",
+			builderConfig:     &Config{BootConfig: bootcommand.BootConfig{BootCommand: []string{"<cOn><leftShiftOn>n<leftShiftOff><cOff>"}}},
+			expectCallSendkey: true,
+			expectedKeysSent:  "shift-c-n",
+			expectedAction:    multistep.ActionContinue,
+		},
```

Comment:
Could we add some test(s) to cover atypical input where things are essentially noop:s? Eg `<cOn><leftShiftOn><cOff>h<leftShiftOff>` should I guess send `H`?

Reply:
of course :+1: